database/migrations/20230613204013-initial-schema.js

```diff
@@ -0,0 +1,100 @@
+'use strict';
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.sequelize.transaction(t => {
+      return Promise.all([
+        queryInterface.createTable('User', {
+          id: {
+            type: Sequelize.DataTypes.BIGINT,
+            autoIncrement: true,
+            primaryKey: true,
+          },
+          username: Sequelize.DataTypes.STRING,
+          email: Sequelize.DataTypes.STRING,
+          password: Sequelize.DataTypes.STRING,
+        }, { transaction: t }),
+        queryInterface.createTable('Team', {
+          id: {
+            type: Sequelize.DataTypes.BIGINT,
+            autoIncrement: true,
+            primaryKey: true,
+          },
+          owner_user_id: {
+            type: Sequelize.DataTypes.BIGINT,
+          },
+          name: Sequelize.DataTypes.STRING,
+        }, { transaction: t }),
+        queryInterface.createTable('Player', {
+          id: {
+            type: Sequelize.DataTypes.BIGINT,
+            autoIncrement: true,
+            primaryKey: true,
+          },
+          name: Sequelize.DataTypes.STRING,
+        }, { transaction: t }),
+        queryInterface.createTable('Year', {
+          id: {
+            type: Sequelize.DataTypes.BIGINT,
+            autoIncrement: true,
+            primaryKey: true,
+          },
+          name: Sequelize.DataTypes.STRING,
+        }, { transaction: t }),
+        queryInterface.createTable('Transaction', {
+          id: {
+            type: Sequelize.DataTypes.BIGINT,
+            autoIncrement: true,
+            primaryKey: true,
+          },
+          is_finalized: {
+            type: Sequelize.DataTypes.BOOLEAN,
+            defaultValue: false,
+            allowNull: false
+          },
+          description: Sequelize.DataTypes.STRING,
+        }, { transaction: t }),
+        queryInterface.createTable('TransactionItem', {
```

Comment:
We might have to go over your vision for `Transaction` or `TransactionItem` so I can see how it's all supposed to work.